### DISCUSSION TITLE
Problem z programowaniem ESP32

### DISCUSSION THREAD

**Discussion by porebskitomasz** [hackerspace-silesia/Smogomierz#161] [Q&A]
Nadszedł czas by "odświeżyć" Smogomierza.
Chciałem go zmajstrować na bazie ESP32
Mam płytke ESP32 DEVKi TV1 ( chip ESP-WROOM-32)
I mam problem aby ją zaprogramować.
Jeśli używam pliku bin i aplikacji FlashDownloadTool 3.9.3 to pokazuje postęp wgrywania, ale po restarcie nie rozgłasza zadnej sieci... (testuje samą płytkę po wgraniu oprogramowania
Jeśli używam pliku Smogomierz.ino i Arduino IDE to wyskakuje błąd kompilacji i nie mogę przejść dalej....
Co mogę zrobić, jak wgrac oprogramowanie do ESP32? (ESP raczej sprawne bo cos innego można wgrać i działa)

**Comment by porebskitomasz** [comment]
Dodam ze programowałem go również zgodnie z opisem NodeMCU PyFlasher

**Reply by eReS72** [reply]
u mnie wygląda wszystko OK po wgraniu oprogramowania, ale też nie widzę sieci... zupełnie nie wiem co z tym poradzić.
próbowałem zrobić ponowny flash ale bez poprawy.

**Reply by porebskitomasz** [reply]
Czekamy może ktoś się odezwie i pomoże....

**Reply by eReS72** [reply]
udało się wgrać soft przez Arduino

**Reply by porebskitomasz** [reply]
No to chyba dzisiaj sądny dzień jest..... Bo mi też od rana wlazło, był problem z jakimiś bibliotekami - przynajmniej u mnie.
Tak na marginesie gdyby ktoś był zainteresowany to mam na zbyciu 2 płytki pcb dla wersji z ESP32
